src/compile/Model.ts

```diff
@@ -233,48 +233,88 @@ export class Model {
     return 30;
   }
 
-  /** @return an object with format and formatType properties. */
+  /**
+   * Returns an object with format and formatType properties.
+   */
   public formatMixins(channel: Channel, format: string) {
```

Comment:
Now that I add util.ts -- should this be moved to util.ts?   

One possible way to make distinction is things `util.ts` might either contains mixins or slightly complicated logic, while methods in model could be something more like helper?